alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/NodeMetrics.java

```diff
@@ -0,0 +1,81 @@
+package eu.xenit.alfred.telemetry.binder;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.ToDoubleFunction;
+
+import io.micrometer.core.instrument.TimeGauge;
+import org.alfresco.repo.domain.node.NodeDAO;
+import org.alfresco.repo.domain.permissions.AclDAO;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.transaction.TransactionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NodeMetrics {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NodeMetrics.class);
+
+    public static final String ALC_PREFIX = "alfresco.alc";
```

Comment:
```suggestion
    public static final String ACL_PREFIX = "alfresco.acl";
```

It's called ACL (Access Control List)